app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<section class="container">
+<div class="header">
+  <%= link_to "<i class='fas fa-arrow-left'></i>".html_safe, categories_path, class: "back" %>
+  <h1>Details</h1>
+  <%= form_for(current_user, url: destroy_user_session_path, method: :delete, html: { class: "nav-link" }) do |f| %>
+    <%= f.button type: :submit, class: "btn-logout logoutBtn" do %>
+      <%= content_tag :i, "", class: "fa-solid fa-right-from-bracket" %>
+    <% end %>
+  <% end %>
+</div>
+<div class="content">
+<h2>Transaction for <%= @category.name %></h2>
+<% if @transactions.any? %>
+  <p class="total">Total Amount: $<%= sprintf('%.2f', @total_amount) %></p>
+  <div >
+    <% @transactions.each do |transaction| %>
+      <ul class="transactions-card">
+        <li>Name: <%= transaction.name %></li>
+        <li>Amount: $<%= sprintf('%.2f', transaction.amount) %></li>
+        <h3><%= button_to 'Delete', category_transaction_path(@category, transaction), method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn-danger' %></h3>
+      </ul>
+    <% end %>
+  </div>
+    <button class="btn addNew"><%= link_to 'Add New', new_category_transaction_path(@category, @transaction)%></button>
+
+<% else %>
+  <p class="msg">No transactions found.</p>
+    <button class="btn addNew"><%= link_to 'Add New', new_category_transaction_path(@category, @transaction)%></button>
+<% end %>
+</div>
```

Comment:
- [ ] Please make sure the list of transactions is presented ordered by the most recent, when I create a new transaction it should be at the top,  good job so far 👍🏿

Reply:
- Well resolved